<?xml version="1.0"?>
<launch>
        <node pkg="omnicopter_ros" type="rc_i2c_driver" name="rc_i2c_driver" output="screen"/>
        <node pkg="omnicopter_ros" type="control_allocation" name="control_allocation" output="screen"/>
        <node pkg="omnicopter_ros" type="attitude_controller" name="attitude_controller" output="screen"/>
        <node pkg="omnicopter_ros" type="position_controller" name="position_controller" output="screen"/>
        <node pkg="omnicopter_ros" type="sonar_driver" name="sonar_driver" output="screen"/>
        <include file="$(find omnicopter_ros)/launch/rtimulib_ros.launch" />
</launch>
